pkgs/development/python-modules/paddlepaddle/default.nix

```
@@ -0,0 +1,90 @@
+{ stdenv
+, lib
```

```suggestion
{ stdenv
, config
, lib
```
Sorry, I forgot this.